pkgs/applications/networking/browsers/ephemeral/default.nix

```
@@ -28,6 +29,11 @@ stdenv.mkDerivation rec {
     sha256 = "1lzcwaczh601kwbx7fzg32nrzlg67asby7p86qy10qz86xf4g608";
   };
 
+  patches = stdenv.lib.optionalString (disableInfobars) [
```

I don't have any particular comment to give about the PR as a whole, just that the parentheses on this line can be removed 🙂
```suggestion
  patches = stdenv.lib.optionalString disableInfobars [
```